website/client/src/components/admin-panel/index.vue

```diff
@@ -0,0 +1,754 @@
+            // these types of items have true/false/null values (all other have integers):
+            const booleanTypes = ['mounts', 'gear'];
+
+            let value;
+            let valueForDisplay;
+            if (booleanTypes.includes(itemType)) {
+              value = ''; // we want no default value in the "Update Items" input box
+              valueForDisplay = 'never owned';
+            } else {
+              value = 0;
+              valueForDisplay = '0 (never owned)';
+            }
+            itemData.push({
+              neverOwned: true,
+              key,
+              name: this.getItemDescription(itemType, key),
+              path: `${basePath}.${key}`,
+              value,
+              valueForDisplay,
+            });
+          }
+        }
+        collatedItemData[itemType] = itemData;
+      });
+      return collatedItemData;
+    },
+    getItemDescription (itemType, key) {
+      // Returns item name. Also returns other info for equipment.
+      const simpleItemTypes = ['eggs', 'hatchingPotions', 'food', 'quests', 'special'];
+      if (simpleItemTypes.includes(itemType) && content[itemType][key]) {
+        return content[itemType][key].text();
+      }
+      if (itemType === 'mounts' && content.mountInfo[key]) {
+        return content.mountInfo[key].text();
+      }
+      if (itemType === 'pets' && content.petInfo[key]) {
+        return content.petInfo[key].text();
+      }
+      if (itemType === 'gear' && content.gear.flat[key]) {
+        const name = content.gear.flat[key].text();
+        const description = this.getGearSetDescription(key);
+        if (description) return `${name} -- ${description}`;
+        return name;
+      }
+      return 'NO NAME - invalid item?';
+    },
+    getGearSetDescription (key) {
+      let setName = this.getGearSetName(key);
+      if (setName === 'special-takeThis') {
+        // no point displaying set details for gear where it's obvious
+        return '';
+      }
+      const klassNames = {
+        healer: 'Healer',
+        rogue: 'Rogue',
+        warrior: 'Warrior',
+        wizard: 'Mage',
+      };
+      const loginIncentivesGear = ['armor_special_bardRobes', 'armor_special_dandySuit', 'armor_special_lunarWarriorArmor', 'armor_special_nomadsCuirass', 'armor_special_pageArmor', 'armor_special_samuraiArmor', 'armor_special_sneakthiefRobes', 'armor_special_snowSovereignRobes', 'back_special_snowdriftVeil', 'head_special_bardHat', 'head_special_clandestineCowl', 'head_special_dandyHat', 'head_special_kabuto', 'head_special_lunarWarriorHelm', 'head_special_pageHelm', 'head_special_snowSovereignCrown', 'head_special_spikedHelm', 'shield_special_diamondStave', 'shield_special_lootBag', 'shield_special_wakizashi', 'shield_special_wintryMirror', 'weapon_special_bardInstrument', 'weapon_special_fencingFoil', 'weapon_special_lunarScythe', 'weapon_special_nomadsScimitar', 'weapon_special_pageBanner', 'weapon_special_skeletonKey', 'weapon_special_tachi'];
```

Comment:
for this list and all the others, it's possible to get the value from `common/content` instead of writing them manually here? it's very error prone if we have to update this list every time something changes

Reply:
It's not really possible to do that, at least not without a lot of extra code (e.g., parsing through all the quests code to pull out data from the rewards), and even then the wiki links would have to be inserted manually.  I don't think the extra code is worth it. 

These are mostly items that won't change at all often or ever, and if new items like these are added to the content, it won't be disastrous if they aren't described like this.

Reply:
okay ! This seemed a bit of anightmare to keep up to date but if you think it won't have to be updated often and automating it would be too complex we can keep it like this